sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/stringExpressions.scala

```diff
@@ -796,6 +812,7 @@ case class MakeValidUTF8(input: Expression) extends RuntimeReplaceable with Impl
 
 }
 
+// TODO: fix
```

Comment:
usually in the Spark repo, we tend to tag TODOs using a corresponding Jira ticket, our your github username

the point is that we make TODOs less random and provide concrete references

Reply:
Sorry, should have put WIP as I should clean-up this pr a bit before review..